### DISCUSSION TITLE
能否把词典配置信息固定在 主文件所在的目录实现真正portable功能

### DISCUSSION THREAD

**Discussion by falltree** [xiaoyifang/goldendict-ng#1894] [General]
目前使用的是23.12.07，安装选择了portable模式，解压到了D盘
但是每次重装系统后，词典设置都不会保留，每次都需要重新设置路径。
我个人半年左右就会重新安装一次windows系统，所以已经多次重装后要重新设置一遍词典。

**Comment by xiaoyifang** [comment]
https://xiaoyifang.github.io/goldendict-ng/topic_portablemode/

你确定是这么设置的吗？原则上配置对了，是不需要重新设置路径的。

**Reply by falltree** [reply]
谢谢，我再按照这个说明试试
